web/src/components/suggestion/Suggestion.vue

```diff
@@ -55,14 +55,15 @@
           </div>
         </div>
         <div v-if="isAuthenticated && role === userRoles.ADMIN" class="suggestion-header-buttons">
-          <assign-user :suggestion="suggestion" class="icon-button" />
           <svg-icon icon-name="more" class="icon-button"><icon-more /></svg-icon>
         </div>
       </div>
 
       <suggestion-content
         :suggestion="suggestion"
         :user-name="userName"
+        :isAuthenticated="isAuthenticated"
```

Comment:
Näitä pitäisi vähän tulevaisuudessa miettiä koska ne pitäisi aina validoida jotenkin että niitä ei käsin huijata

Reply:
check